src/test/java/chess/domain/piece/PieceTest.java

```diff
@@ -0,0 +1,74 @@
+package chess.domain.piece;
+
+import chess.domain.piece.strategy.RookMovementStrategy;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.DisplayNameGeneration;
+import org.junit.jupiter.api.DisplayNameGenerator;
+import org.junit.jupiter.api.Test;
+
+import static chess.domain.piece.position.PiecePosition.of;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+@SuppressWarnings("NonAsciiCharacters")
+@DisplayNameGeneration(DisplayNameGenerator.ReplaceUnderscores.class)
+@DisplayName("Piece 은")
+class PieceTest {
+
+    private final Color myColor = Color.WHITE;
+    private final Color enemyColor = Color.BLACK;
+
+    @Test
+    void 경유지탐색_시_같은_위치면_예외() {
+        // given
+        Piece myPiece = new Piece(of(1, 'a'), new RookMovementStrategy(myColor));
```

Comment:
```suggestion
        Piece myPiece = new Piece(PiecePosition.of(1, 'a'), new RookMovementStrategy(myColor));
```
어떤 클래스의 of인지 명시해주는게 좋을것 같습니다

Reply:
반영했습니다